rootfs/api/tests/test_release.py

```diff
@@ -320,11 +320,13 @@ def test_release_rollback_failure(self, mock_requests):
             with mock.patch('api.models.Release._delete_release_in_scheduler') as mock_kube:
                 # instead of full request mocking, fake it out in a simple way
                 class Response(object):
-                    pass
+                    def json(self):
+                        return '{}'
 
                 response = Response()
                 response.status_code = 404
-                kube_exception = KubeHTTPException(response=response)
+                response.reason = "Not Found"
```

Comment:
Ah! hadn't thought of that but I did give it a go just now and it won't work because of `AttributeError: 'HttpResponseNotFound' object has no attribute 'reason'` 

`KubeException` is expecting a `python-requests` response object